### DISCUSSION TITLE
Present the "Quick start" page before the "Devtools" page in the docs

### DISCUSSION THREAD

**Discussion by AntonNiklasson** [TanStack/query#1862] [Ideas]
I've never used React Query before, and was just going through the docs. Looks like great docs btw!

But I noticed that the flow takes me from "Getting started" on the homepage, through to "Installation" and then "Devtools". At this point I have no idea what the `queryClient` is, it's introduced later. To me, it looks like "Quick start" makes more sense before the dev tools page. I'd love to PR it if it makes sense to you.

I'm excited to dig into React Query 🥰

**Comment by TkDodo** [comment]
yeah, we could definitely move it behind the `QuickStart` (I'd approve a PR of this :) - but it also doesn't explain things like queryClient there. The example in `Getting started` also uses QueryClient. most things are explained in the `Guides & Concept` section.

**Reply by AntonNiklasson** [reply]
Okey, great. I'll open a PR for this, and we can discuss the details further there.

EDIT: https://github.com/tannerlinsley/react-query/pull/1865
